pkgs/by-name/hypercore/dream2.nix

```
@@ -18,10 +18,13 @@ rec {
       owner = "holepunchto";
       repo = "hypercore";
       rev = "v${version}";
-      sha256 = "sha256-NtDd6wPuTbSvXW0NZr+XFMoeTqUlrb/C6oOu3uzBHuw=";
+      sha256 = "sha256-RTYeaJQ88m2be8pxuumVrDde1L5u98NeTHkZ9EN59wI=";
     };
 
     doCheck = false;
```

We do want the tests enable. Only this small change and I think we're good.

```suggestion
    doCheck = true;
```